http-clients/src/main/java/org/triplea/http/client/JsonEncoder.java

```diff
@@ -15,6 +15,7 @@
 public class JsonEncoder implements Encoder {
   private static final GsonEncoder gsonEncoder = new GsonEncoder();
 
+  @SuppressWarnings("deprecation")
```

Comment:
The deprecated method was introduced in a recent PR. It's not clear what the alternative usage would be, perhaps I have not looked into it enough. Regardless, the encode logic is inspired/copy+pasted from the default feign encoder (thank you open source!) which uses a deprecated method. It seems to work perfectly fine for now, hence I'm not terribly concerned yet about the deprecated API usage and am just suppressing it for now.